src/pages/Overview/OverviewPage.tsx

```diff
@@ -84,7 +84,7 @@ const gridStyleList = style({
   marginTop: '0px'
 });
 
-const cardGridStyle = style({ borderTop: '2px solid #39a5dc', textAlign: 'center', marginTop: '0px' });
+const cardGridStyle = style({ borderTop: '2px solid #39a5dc', textAlign: 'center', marginTop: '10px' });
```

Comment:
I'd recommend

```
const cardGridStyle = style({ borderTop: '2px solid #39a5dc', textAlign: 'center', marginTop: '0px', marginBottom: '10px' });
```

Then the style from the overview and the list pages would keep same top margin:
![image](https://user-images.githubusercontent.com/1662329/133767273-7480b120-05e2-486b-9149-25270309eaac.png)
![image](https://user-images.githubusercontent.com/1662329/133767292-d955c5cb-0951-4d2e-845c-20b9d1f4c92d.png)